backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/generics/report_orgblock.jrxml

```diff
@@ -90,7 +91,7 @@ FROM
 				<textFieldExpression><![CDATA[$R{vataxid} + " " + $F{vataxid}]]></textFieldExpression>
 			</textField>
 			<textField isBlankWhenNull="true">
-				<reportElement uuid="382642ff-3a69-4cd5-ae1c-f374c3c8226d" x="37" y="22" width="142" height="12"/>
+				<reportElement x="0" y="22" width="142" height="12" uuid="382642ff-3a69-4cd5-ae1c-f374c3c8226d"/>
```

Comment:
Like above!
Do not change this subreport.
I will affect all docs. You shall do adjustments from your own report.